unixfs/tar/writer.go

```diff
@@ -31,12 +32,18 @@ func DagArchive(ctx cxt.Context, nd *mdag.Node, name string, dag mdag.DAGService
 	// use a buffered writer to parallelize task
 	bufw := bufio.NewWriterSize(pipew, DefaultBufSize)
```

Comment:
This is not necessary if https://github.com/ipfs/go-ipfs/blob/master/unixfs/tar/writer.go#L123 is replaced with `dagr.WriteTo(w.TarW)`?